.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/ckan-php-manager
+    docker:
+      - image: circleci/php:7.1
+    steps:
+      - checkout
+      - run:
+          name: Install Composer
+          command: |
+            php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
```

Comment:
FYI, it looks like the image already includes composer, so all you really need to do is run phpunit

```
$ docker run --rm -it circleci/php:7.1 composer --version
Composer version 1.9.1 2019-11-01 17:20:17
```